### ISSUE TITLE
mlagents_envs shutdown with return code 0

### ISSUE THREAD

**Issue by Kasidit0052** [Unity-Technologies/obstacle-tower-env#125]
Hi, I'm trying to launch obstacle tower environment. However, the obstacle env window close. This is the log

    INFO:mlagents_envs.environment:Environment shut down with return code 0.

I'm using mlagents 0.19.0 and mlagents-env 0.19.0. Does obstacle-tower-env support mlagents-env 0.19?, since I have encountered an error about mlagents-env earlier during environment installation.
Thank you

**Comment by awjuliani** [comment]
Hi @Kasidit0052, can you share what platform you are running Obstacle Tower on? The latest Obstacle Tower was tested with ML-Agents release 3, which corresponds to the python package 0.17. I don't imagine there would be an incompatibility with 0.19, but just in case, it may be worth testing with the earlier version to be sure. Another good way to check whether there is an issue with the binary of the python api is to try running the binary by itself, which should launch the environment in a human-playable "game" mode.

**Comment by Kasidit0052** [comment]
Thank you for your response, I am running Obstacle Tower on Mac OS Catalina 10.15.6. Now, I'm trying to download binary separately for Mac OS X. However, I'm not sure about the problem since The Obstacle Tower application which I have just downloaded work fine.

**Comment by awjuliani** [comment]
Just to be clear, you are saying that the binary you downloaded yourself works fine, but the one auto-downloaded by the gym wrapper is giving you issues?

**Comment by Kasidit0052** [comment]
Yes, I have solved the problem by configuring the binary path to the newer binary ( download by myself).
